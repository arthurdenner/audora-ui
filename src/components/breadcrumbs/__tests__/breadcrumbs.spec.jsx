import React from 'react';
import { render } from 'enzyme';
import Breadcrumbs from '../breadcrumbs';

describe('Breadcrumbs', () => {
  it('renders correctly', () => {
    const items = [
      { text: 'Pages', href: '#' },
      { text: 'Home', href: '#' },
      { text: 'Breadcrumbs', href: '#' },
    ];

    const wrapper = render(<Breadcrumbs items={items} />);
    expect(wrapper).toMatchSnapshot();
  });
});
