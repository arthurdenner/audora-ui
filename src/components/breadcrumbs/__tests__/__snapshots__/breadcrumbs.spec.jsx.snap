// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<div
  class="sc-bwzfXH jIEWCi"
>
  <div
    class="sc-bxivhb cDFCAd"
  >
    <a
      class="sc-ifAKCX gSzACM"
      href="#"
    >
      Pages
    </a>
  </div>
  <div
    class="sc-bxivhb cDFCAd"
  >
    <div
      class="sc-htpNat TryqS"
    >
      /
    </div>
    <a
      class="sc-ifAKCX gSzACM"
      href="#"
    >
      Home
    </a>
  </div>
  <div
    class="sc-bxivhb cDFCAd"
  >
    <div
      class="sc-htpNat TryqS"
    >
      /
    </div>
    <a
      class="sc-ifAKCX gSzACM"
      href="#"
    >
      Breadcrumbs
    </a>
  </div>
</div>
`;
